install/datadog/install.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+# Copyright 2017 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+SOURCE_DIR=$(dirname $0)
+HAVE_KEYS=0
+
+function prompt_if_unset() {
+  local name=$1
+  local tmp
+  while [[ "${!name}" == "" ]]; do
```

Comment:
Not all shells support this syntax, I think a more backward compatible alternative is
```bash
while [ -n "$name" ]; do
```

which just checks that `$name` is non-empty.